.pipelines/e2e.yaml

```diff
@@ -13,12 +13,7 @@ pr:
       - .pipelines/e2e.yaml
       - .pipelines/templates/e2e-template.yaml
       - e2e
-      - parts/linux
```

Comment:
we will want to trigger on changes to files within these directories, as they contain the core templates used by agentbakersvc to generate CSE/customdata